dapp_user/user.py

```diff
@@ -40,22 +42,60 @@ def _set_user_data(self, user_data):
             print(repr(e))
             raise e
 
-    def get_wallet_details(self, user_data):
+    def get_wallet_details(self, user_data, org_id, group_id):
         """ Method to get wallet details for a given username. """
         try:
             username = user_data['authorizer']['claims']['email']
-            search_data = self.repo.execute(
-                "SELECT * FROM wallet WHERE username = %s", username)
-            self.obj_utils.clean(search_data)
-            return search_data
+            wallet_data = self.repo.execute(
+                "SELECT UW.address, UW.is_default, W.type, W.status "
+                "FROM user_wallet as UW JOIN wallet as W ON UW.address = W.address WHERE UW.username= %s", username)
+            self.obj_utils.clean(wallet_data)
+
+            wallet_response = {
+                "username": username,
+                "org_id": org_id,
+                "group_id": group_id,
+                "wallets": []
+            }
+            for record in wallet_data:
+                wallet = dict()
+                wallet.update(record)
```

Comment:
whenever function becomes big try to break into small functions to enhance readability